data/frame_type.go

```diff
@@ -88,7 +108,8 @@ func (p FrameType) IsTimeSeries() bool {
 	case
 		FrameTypeTimeSeriesWide,
 		FrameTypeTimeSeriesLong,
-		FrameTypeTimeSeriesMany:
+		FrameTypeTimeSeriesMany,
+		FrameTypeTimeSeriesLabeled:
```

Comment:
@kylebrandt -- would returning "true" for this break anything?  IE does anything expect to know how to work with it if it is set?